src/java.base/share/classes/java/io/InputStreamReader.java

```diff
@@ -35,7 +35,7 @@
  * An InputStreamReader is a bridge from byte streams to character streams: It
  * reads bytes and decodes them into characters using a specified {@link
  * java.nio.charset.Charset charset}.  The charset that it uses
- * may be specified by name or may be given explicitly, or the platform's
+ * may be specified by name or may be given explicitly, or the
  * {@link Charset#defaultCharset() default charset} may be accepted.
```

Comment:
"may be accepted" seems like the API has some choice in the matter. 
Perhaps "accepted" -> "used".
And in other classes below if there's a suitable replacement.